src/scripts/activate-user.ts

```diff
@@ -130,7 +130,7 @@ export const activateUser =
       TE.tapIO((user) =>
         Console.info(
           `Activated [${user.id}] [${user.email}].${
-            sendEmail ? ' Email has been sent to [${user.email}].' : ''
+            sendEmail ? ` Email has been sent to [${user.email}].` : ''
```

Comment:
Sorry, I didn't notice it until I tested the code. But the second square bracket is on the outside of the curly bracket.